content/en/docs/getting-started/_index.html

```diff
@@ -0,0 +1,266 @@
++++
+title="Getting Started"
+linkTitle="Getting Started"
+description="How to get up and running with Jenkins X"
+aliases="/getting-started"
+
+type= "docs"
+menu="Docs"
+weight="1"
+layout="home"
+
++++
+
+
+<h1>Get started</h1>
```

Comment:
By using HTML over Markdown, we are losing the right-hand navigation over the section headings. It would be nice if the right hand would read "Step 1 Step2 ..."

TBH, I still believe it should be not too hard to use Markdown here and put the tab stuff into a Hugo shortcode. See also https://discourse.gohugo.io/t/how-was-the-tabbed-content-interface-done-in-hugo-docs/11795

Reply:
There is still the "Setup" page - http://jx-docs.jx-jenkins-x-jx-docs-pr-2615.jenkins-x.live/docs/getting-started/setup/

Really this page should be removed/replaced by this content

Reply:
I have removed the old setup page.